client/controllers/EnemyBirdsController.ts

```diff
@@ -1,24 +1,31 @@
-import EnemyBird from "../components/EnemyBird";
+import { EVENTS } from "../../configs/events";
+import EnemyBird, { EnemyBirdAttributes } from "../view/birds/EnemyBird";
 
 // EnemyBirdsFactory creates new EnemyBirds and updates existing ones
-class EnemyBirdsFactory {
+class EnemyBirdsController {
+  private socket: SocketIOClient.Socket;
+
   private storedBirds: {
     [k: string]: EnemyBird;
   } = {};
 
   constructor(socket: SocketIOClient.Socket) {
-    socket.on("otherBird", (bird: { id: string; y: number; momentum: number }) => {
+    this.socket = socket;
+
+    this.socket.on(EVENTS.OTHER_BIRD, (bird: EnemyBirdAttributes) => {
```

Comment:
It's fine, but in the future we can think about creating some kind of a socket events handler - to manage all events related actions (not sure if it's good idea, just a proposal)